### PR TITLE
Update cluster reporting script for project submissions/recommendations

### PR DESCRIPTION
**Pull request by rmoneys**
Updates our job run reporting script to create submissions and recommendations instead of predictions.

### CODE REVIEW

**Review by singhals — DISMISSED** [review]

**Comment by rmoneys** [comment]
Success with `auto_apply_recs` not set: https://dbc-d95d06ca-1d00.cloud.databricks.com/?o=656201176161048#job/217634515132762/run/120298521562066

File: integration/send-logs-gradient.py [hunk 1]
```diff
@@ -52,14 +54,42 @@
...
+
+    if response.error:
+        raise RuntimeError(str(response.error))
+
+    recommendation_id = response.result
+
+    response = projects.wait_for_recommendation(project["id"], recommendation_id)
+
+    if response.error:
+        raise RuntimeError(str(response.error))
```
Comment (gorskysd) [inline integration/send-logs-gradient.py:86]: Do we not want to combine these operations into one method in the library? You might just be saving that as followup work.

Comment (gorskysd) [inline integration/send-logs-gradient.py]: Do we not want to combine these operations into one method in the library? You might just be saving that as followup work.

Comment (rmoneys) [reply integration/send-logs-gradient.py]: Yeah, it's probably better to have a `create_and_apply_recommendation` function in the library for everything in this "auto_appy_recs" clause - if only to minimize the code that is further away from our control, i.e. in a notebook somewhere. Is that what you're thinking?

Comment (rmoneys) [reply integration/send-logs-gradient.py]: Yeah - I'll put all of this into something like `record_run_ml`

Comment (rmoneys) [reply integration/send-logs-gradient.py]: Addressed in this PR, https://github.com/synccomputingcode/client_tools/pull/27, which should supersede this one.

**Pull request closed without merging**